README.md

```diff
@@ -78,6 +117,8 @@ optional arguments:
                         Filter metrics (default: disabled)
   -l {info,warn,error}, --log {info,warn,error}
                         Logging level (default: info)
+  -mp PORT, --metrics-port PORT   
```

Comment:
Let's not use "single dash multiple letters" arguments. It violates the rule:
https://serverfault.com/questions/387935/whats-the-difference-betwen-the-single-dash-and-double-dash-flags-on-shell-comm
In this case it is ok to only have `--metrics-port` without short version.